components/todo/Header.vue

```diff
@@ -40,15 +43,50 @@ export default {
 }
 </script>
 <style scoped lang="scss">
-.header {
+@import '@/assets/css/variables';
+@import '@/assets/css/mixins';
+
+header {
+  @include padding(20px, $padding-breakpoints);
   display: flex;
   justify-content: space-between;
   align-items: center;
+  height: $header-height;
+  background-color: #fff;
 }
 
 .header__left {
   display: flex;
   align-items: center;
   gap: 10px;
 }
+
+.todo__title {
+  width: 150px;
+  font-size: 36px;
+  font-weight: 700;
+  line-height: 42.19px;
+  color: #7a8dfd;
+}
+
+.todo__language {
+  width: 180px;
+  height: 36px;
+  padding: 0 10px;
+  border: 1px solid #e5e5e5;
+  outline: none;
+  border-radius: 5px;
+  font-size: 16px;
+  line-height: 19.69px;
+  color: #32394b;
+  background-color: #fff;
+  cursor: pointer;
+}
+
+@media screen and (max-width: 768px) {
```

Comment:
nitpicking: we could use variables for breakpoints

Reply:
resolved. Bhai